libr/anal/p/anal_mips_cs.c

```diff
@@ -602,9 +603,15 @@ static int analop_esil(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len
 		}
 		break;
 #endif
-	default:
-		return -1;
+		default:
+			return -1;
+		}
 	}
+	if (strstr (r_strbuf_get (&op->esil), "$$")) {
+		char addr_str[19];	// "0x" + 64/4 + "\x00"
+		sprintf (addr_str, "0x%"PFMT64x, addr);	//yes, it's fine to use sprintf in this particular case
```

Comment:
```suggestion
		snprintf (addr_str, sizeof(addr_str), "0x%"PFMT64x, addr); //yes, it's fine to use sprintf in this particular case
```

Reply:
i see no point in snprintf here, since the "0x%"PMFT64x cannot exceed length of 18